app/models/measurement_details.rb

```diff
@@ -0,0 +1,213 @@
+# rubocop:disable Metrics/ClassLength
```

Comment:
You can move this to the same line as the class definition to scope it just to this class, otherwise add a rubocop:enable at the end of the file. Not sure if they globally disable/enable, but ever example from rubocop team has them paired.